docs/chart-review.md

```diff
@@ -8,7 +8,9 @@ nav_order: 20
 
 # Chart Review
```

Comment:
I kept this documentation file as being *about* Chart Review, but everything inside describes the upload-notes command and what I call "upload mode".

I figured Chart Review was what the user's brain would say they are trying to do, and this doc describes the Cumulus ETL command that helps with that.

But open to notes on that - I could buy confusion around it.

Reply:
i think chart review is the business logic name for this, so i buy it.